addon/-private/system/relationships/state/relationship.js

```diff
@@ -346,7 +346,7 @@ export default class Relationship {
     this.store._updateRelationshipState(this);
   }
 
-  updateLink(link) {
+  updateLink(link, initial) {
```

Comment:
oooooh, that this arg was already present implies that at some point we'd thought of this case and lost it